modules/cnf-nrop-support-schedulable-resources.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+:_mod-docs-content-type: CONCEPT
+[id="cnf-numa-resource-operator-support-scheduling-cp_{context}"]
+=  NUMA Resources Operator support for schedulable control-plane nodes
+
+[role="_abstract"]
+You can enable schedulable control plane nodes to run user-defined pods, effectively turning them into hybrid control plane and worker nodes, which is especially beneficial in resource-constrained environments, such as compact clusters. When enabled, the NUMA Resources Operator can apply its topology-aware scheduling to these nodes for guaranteed workloads, ensuring pods are placed according to the best NUMA affinity.
+
+Traditionally, control plane nodes in {product-title} are dedicated to running critical cluster services. By enabling schedulable control plane nodes, you allow user-defined pods to be scheduled on these nodes. This effectively turns the nodes into a hybrid of control plane and worker nodes, which is especially useful in resource-constrained environments, such as compact clusters. For example, when you install a three-node cluster, the three control plane machines also act as compute machines. This configuration is part of the installation process.
```

Comment:
Curious sense of deja vu here in the phrase "....effectively turns the nodes into a hybrid of control plane and worker nodes, which is especially useful in resource-constrained environments, such as compact clusters." 

More or less a repetition of line 9 above.